app/controllers/people_controller.rb

```diff
@@ -0,0 +1,19 @@
+class PeopleController < ApplicationController
+
+  def index
+    @all_people = Person.all
+    render :index
+  end
+
+  def show
+    @person = Person.find(params[:id])
+    render :show
```

Comment:
The way Rails works, if you name the view template the same as the action, you don't need to call `render` — it will do it for you. So you could actually skip the render lines in all three actions here.